go.mod

```diff
@@ -5,18 +5,16 @@ go 1.21
 require (
 	github.com/denisenkom/go-mssqldb v0.12.3
 	github.com/google/uuid v1.6.0
+	github.com/pkg/errors v0.9.1
```

Comment:
Do you have a particular reason to use `pkg/errors` here?

I was hoping we could gradually phase this package out and rather use the error package from the standard library. As far as I can see you only use functions that are also available in the standard libraries `errors` package. `Is` and `As`. The `Errorf` method can be replaced with `fmt.Errorf`.

The thing is that this project was archived in 2021 when the standard libraries error package was improved. The only reason, afaik, we kept using `pkg/errors` is that it allows attaching the stack trace easily. (Which I personally think should not be required, but never really managed to convince everyone.).

Reply:
Nice catch!  I removed pkg/errors now.